src/main/webapp/scripts/usermanagement/auth/principal.service.js

```diff
@@ -61,7 +68,17 @@ angular.module('metadatamanagementApp').factory(
             $rootScope.$broadcast('stop-ignoring-401');
             _identity = account.data;
             _authenticated = true;
-            deferred.resolve(_identity);
```

Comment:
@es-flavia As mentioned I do not get why we need to block here...

Reply:
Done